packages/backend-core/src/db/Replication.ts

```diff
@@ -27,7 +27,7 @@ class Replication {
           return resolve(info)
         })
         .on("error", function (err) {
-          throw new Error(`Replication Error: ${err}`)
+          throw err
```

Comment:
This caused an error in a test while I was working on this change, and I moved to throwing the raw error in order to see the full stack trace. Decided to leave it as is because I feel this is more useful.